apache/couchdb-fauxton
# [partitioned-dbs] Support create and list databases

## Overview

First in a series of PRs that will be submitted in support of the new user-defined partitioned databases feature (https://github.com/apache/couchdb/pull/1605).

To provide backwards compatibility, Fauxton will detect if the feature is enabled on the server and only display the new GUI elements based on this check.

Summary:
 - Detect and enable support if the feature is available on the server
 - Add 'Partitioned' checkbox when creating a new database
 - Show 'Partitioned' column in the database list

## Testing recommendations

 - Create a new database and check the 'Partitioned' checkbox
 - Verify the new database is listed and its 'Partitioned' column reads 'Yes'

## Related Pull Requests

https://github.com/apache/couchdb/pull/1605

## Checklist
